.github/workflows/build.yml

```diff
@@ -1,52 +1,48 @@
 # GitHub Actions Workflow is created for testing and preparing the plugin release in the following steps:
-# - validate Gradle Wrapper,
-# - run 'test' and 'verifyPlugin' tasks,
-# - run Qodana inspections,
-# - run 'buildPlugin' task and prepare artifact for the further tests,
-# - run 'runPluginVerifier' task,
-# - create a draft release.
+# - Validate Gradle Wrapper.
```

Comment:
Merged all of this from the upstream template so we're caught up with best practices for how to build/release plugins. The only major delta for us is the setup-go task.